src/main/resources/assets/prefab/lang/en_us.json

```diff
@@ -250,6 +250,8 @@
   "prefab.gui.bulldozer_desc": "This item will clear out a 16x16x16 space and provide the drops from each block.",
   "prefab.gui.compressed_chest_desc": "Used in the recipes for structures, not for direct storage",
   "prefab.gui.sickle_desc": "Cut grass, flowers, and leaves with ease!",
+  "prefab.gui.button_desc.1": "Left Click to toggle",
```

Comment:
This assumes a right-handed mouse configuration.
"Click for next option"
"Secondary Click for previous option"

Reply:
Don't think you can check if the mouse is left-handed or right-handed 
Doesn't even mater really because they aren't going to know that they're actually toggling in reverse
The purpose of the tooltip is just to say that you can toggle the button + that you can toggle it backwards by using right click